### PR TITLE
PR for GitHUb Flow

### PR DESCRIPTION
**Pull request by ghost**
@PinoEire An bhfuil Gaeilge agat Pino?

### CODE REVIEW

File: eoinoconghaile-demo-file.md [hunk 1]
```diff
@@ -0,0 +1 @@
+## Demo File a mhac
```
Comment (pvencel) [inline eoinoconghaile-demo-file.md:1]: Looks good

Comment (PinoEire) [inline eoinoconghaile-demo-file.md:1]: Nope, looks like a :bug:   --> :trollface:

**Comment by ericcurtin** [comment]
You merged this piece of crap in. WHAT WERE YOU THINKING ? :)

**Comment by ghost** [comment]
@ericcurtin I know...I'll get fired

**Pull request merged**